server/src/test/java/io/whitefox/core/services/S3FileSignerTest.java

```diff
@@ -0,0 +1,53 @@
+package io.whitefox.core.services;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.whitefox.api.deltasharing.S3TestConfig;
+import io.whitefox.core.*;
+import jakarta.inject.Inject;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.commons.util.StringUtils;
+
+@QuarkusTest
```

Comment:
Do we need this to be a quarkus test?

Reply:
so we need it because of S3TestConfig, but is it worth to make the test 1-2 seconds slower only to get 3 environment variables?

Reply:
Yes because it is an integration test in that it signs a file on s3. it is an integration test because I need to initialize an `S3FileSigner`, to do that I need whitefox access keys, and to get them I need the application to be up.

On the other hand, if I turn it into a unit test by using fake credentials and mocking the `S3FileSigner`, I would not actually test anything.

Reply:
yes it's worth since this test bottleneck is I/O setting up quarkus is fast enough to make it not noticeable